app/views/admin/hearings/_form.html.erb

```diff
@@ -15,47 +15,62 @@
     <%= form.label :jurisdictionType, 'Jurisdiction Type' %>
     <%= form.select :jurisdictionType, Hearing::JURISDICTION_TYPES %>
   </div>
+
   <div class="field">
     <%= form.label :reportingRestrictionReason, 'Reporting Restriction Reason' %>
     <%= form.text_field :reportingRestrictionReason %>
   </div>
+
   <div class="field">
     <%= form.label :court_centre_id, 'Court Centre' %>
     <%= form.select :court_centre_id, HmctsCommonPlatform::Reference::CourtCentre.all.collect{ |cc| [cc.oucode_l3_name, cc.id ] } %>
   </div>
+
   <div class="field">
     <%= form.label :hearingLanguage, 'Hearing Language' %>
     <%= form.select :hearingLanguage, Hearing::LANGUAGES %>
   </div>
+
   <div class="field">
     <%= form.label :hasSharedResults, 'Has shared results' %>
     <%= form.check_box :hasSharedResults %>
   </div>
+
   <%= form.fields_for :hearing_type do |hearing_type| %>
     <div class="field">
       <%= hearing_type.label :description, 'Hearing Type description' %>
       <%= hearing_type.text_field :description %>
     </div>
   <% end %>
+
   <%= form.fields_for :hearing_days do |hearing_day| %>
     <div class="field">
       <%= hearing_day.label :sittingDay %>
       <%= hearing_day.text_field :sittingDay %>
     </div>
+
     <div class="field">
       <%= hearing_day.label :listedDurationMinutes %>
       <%= hearing_day.text_field :listedDurationMinutes %>
     </div>
   <% end %>
+
   <div class="field">
     <%= form.label :isEffectiveTrial %>
     <%= form.check_box :isEffectiveTrial %>
   </div>
+
   <div class="field">
     <%= form.label :isBoxHearing %>
     <%= form.check_box :isBoxHearing %>
   </div>
 
+  <h3>Cracked Ineffective Trial</h3>
+
+  <div class="field">
+    <%= form.select(:cracked_ineffective_trial_id, CrackedIneffectiveTrial.all.collect { |cit| [cit.description, cit.id] }, include_blank: true) %>
```

Comment:
Does this dropdown work for you locally?
CrackedIneffectiveTrial.all is an empty array for me

Reply:
@arthurashman running `rake task mock:demodata:load` will create a cracked trial record in the database, and the dropdown will have it.

https://github.com/ministryofjustice/hmcts-common-platform-mock-api/blob/master/lib/tasks/demodata.rake